drivers/scheduler/k8s/k8s.go

```diff
@@ -311,68 +314,140 @@ func (k *K8s) ParseSpecs(specDir, storageProvisioner string) ([]interface{}, err
 		return nil, err
 	}
 
+	logrus.Infof("fileList: %v", fileList)
 	var specs []interface{}
 
 	splitPath := strings.Split(specDir, "/")
 	appName := splitPath[len(splitPath)-1]
 
 	for _, fileName := range fileList {
-		file, err := ioutil.ReadFile(fileName)
+		isHelmChart, err := k.IsAppHelmChartType(fileName)
 		if err != nil {
 			return nil, err
 		}
+		if !isHelmChart {
+			file, err := ioutil.ReadFile(fileName)
+			if err != nil {
+				return nil, err
+			}
 
-		var customConfig scheduler.AppConfig
-		var ok bool
-
-		if customConfig, ok = k.customConfig[appName]; !ok {
-			customConfig = scheduler.AppConfig{}
-		}
+			var customConfig scheduler.AppConfig
+			var ok bool
 
-		var funcs = template.FuncMap{
-			"Iterate": func(count int) []int {
-				var i int
-				var Items []int
-				for i = 1; i <= (count); i++ {
-					Items = append(Items, i)
-				}
-				return Items
-			},
-		}
+			if customConfig, ok = k.customConfig[appName]; !ok {
```

Comment:
@diptiranjanpx can you confirm why we are deleting this code block here.
@bhavana-ar